### PR TITLE
Replace DMT browser with new Tree

### LINKED ISSUE #1078: Update DMT tree browser to use "SimpleTree"

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- Replaces the "main tree browser thing" with a component which uses the "new SimpleTree" (Editor.tsx:L79)
- Deletes a ton of code related to "index" both frontend and API
- Some regression on available contextMenu actions

## Why is this pull request needed?
- Tech debt

## Issues related to this change:
closes #1078

### CODE REVIEW

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx [hunk 1]
```diff
@@ -1,137 +1,96 @@
...
+      <MenuItem key={'new-root-package'} onClick={() => {}}>
+        New package
+      </MenuItem>
+    )
+  }
+
+  // Packages get a "new folder"
+  if (node.type == BlueprintEnum.PACKAGE) {
+    menuItems.push(
+      <MenuItem key={'new-package'} onClick={() => {}}>
```
Comment (KristianKjerstad) [inline web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:49]: functionality for new folder onClick is missing

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx [hunk 2]
```diff
@@ -1,137 +1,96 @@
...
+      </MenuItem>
+    )
+  }
+
+  // Everything besides dataSources and folders can be viewed
+  if (!['dataSource', BlueprintEnum.PACKAGE].includes(node.type)) {
+    menuItems.push(
+      // @ts-ignore
+      <MenuItem key={'view'} onClick={() => ViewAction(node)}>
+        View link
```
Comment (KristianKjerstad) [inline web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:60]: Maybe "View/Open in new tab" makes more sense?

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx [hunk 3]
```diff
@@ -1,137 +1,96 @@
...
+  // dataSources get a "new root package"
+  if (node.type === 'dataSource') {
+    menuItems.push(
+      <MenuItem key={'new-root-package'} onClick={() => {}}>
+        New package
+      </MenuItem>
+    )
+  }
+
+  // Packages get a "new folder"
```
Comment (KristianKjerstad) [inline web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:46]: maybe we also new "new blueprint" and "new entity"?

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx (hunk 1 above)
Comment (soofstad) [reply web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:49]: Issue created

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx (hunk 3 above)
Comment (soofstad) [reply web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:46]: issues created

File: web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx [hunk 4]
```diff
@@ -1,137 +1,96 @@
...
+        View link
+      </MenuItem>
+    )
+  }
+
+  // Everything besides dataSources can be deleted
+  if (node.type !== 'dataSource') {
+    menuItems.push(
+      <MenuItem key={'delete'} onClick={() => DeleteAction(node)}>
+        Delete
```
[the remark this answers is not in the document]
Comment (soofstad) [reply web/packages/plugins/apps/dmt-app/src/components/context-menu/ContextMenu.tsx:69]: Done

File: web/packages/plugins/apps/dmt-app/src/pages/editor/Editor.tsx [hunk 5]
```diff
@@ -9,16 +7,19 @@ import { LayoutComponents } from '../../context/dashboard/useLayout'
...
 
-export const Wrapper = styled.div`
-  width: 100%;
-  padding-right: 20px;
+export const TreeWrapper = styled.div`
+  width: 25%;
+  margin-right: 10px;
+  background-color: whitesmoke;
+  height: 100vh;
+  border: black solid 1px;
```
Comment (eoaksnes) [inline web/packages/plugins/apps/dmt-app/src/pages/editor/Editor.tsx:21]: Do we want a border around the tree?

**Review by eoaksnes — APPROVED** [review]
Nice

**Review by KristianKjerstad — APPROVED** [review]

File: web/packages/plugins/apps/dmt-app/src/pages/editor/Editor.tsx (hunk 5 above)
Comment (soofstad) [reply web/packages/plugins/apps/dmt-app/src/pages/editor/Editor.tsx:21]: Agreed. Removed borders

**Pull request merged**